src/FnStream.php

```diff
@@ -86,7 +86,11 @@ public static function decorate(StreamInterface $stream, array $methods)
 
     public function __toString()
     {
-        return call_user_func($this->_fn___toString);
+        try {
+            return call_user_func($this->_fn___toString);
+        } catch (\Throwable $e){
+            return '';
```

Comment:
I wonder if this should also raise an error.

Reply:
Done for all resolving.